.gitignore

```diff
@@ -0,0 +1,2 @@
+/.idea
```

Comment:
Heel goed dat je .idea in je gitignore zet, maar toch staat je .idea folder op github.